src/common/actions/snaps.js

```diff
@@ -9,16 +9,15 @@ export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
 export const FETCH_SNAPS_SUCCESS = 'FETCH_SNAPS_SUCCESS';
 export const FETCH_SNAPS_ERROR = 'FETCH_SNAPS_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps() {
```

Comment:
Should this be the responsibility of the client or the API?

If we add organisation support to the application, should the API not supply all snaps the user has access to? It's difficult for me to judge without knowing what organisation support will look like.